### PR TITLE
GH-5064: added docker makefile for Jetty 9

### PR DESCRIPTION
**Pull request by barthanssens**
GitHub issue resolved: #5064 

Briefly describe the changes proposed in this PR:
- add docker makefile for Jetty 9
- updated docker makefile for Tomcat 9 (instead of 8.5)
- updated e2e tests to run the tests for both Jetty and Tomcat
- minor updates of documentation
- added leading slash in overview.jsp to redirect to correct path (Jetty)

----
PR Author Checklist (see the [contributor guidelines](https://github.com/eclipse/rdf4j/blob/main/CONTRIBUTING.md) for more details):

 - [x] my pull request is [self-contained](https://rdf4j.org/documentation/developer/merge-strategy/#self-contained-changes-pull-requests-and-commits)
 - [x] I've added tests for the changes I made
 - [x] I've applied [code formatting](https://github.com/eclipse/rdf4j/blob/main/CONTRIBUTING.md#code-formatting) (you can use `mvn process-resources` to format from the command line)
 - [x] I've [squashed](https://rdf4j.org/documentation/developer/squashing) my commits where necessary 
 - [x] every commit message starts with the issue number (GH-xxxx) followed by a meaningful description of the change

### CODE REVIEW

**Comment by hmottestad** [comment]
The end-to-end tests probably just build the default dockerfile. Could you have it run against both the default/current and the jetty one?

**Comment by barthanssens** [comment]
Ah, good thinking, forgot about that

**Comment by barthanssens** [comment]
Workbench seems to work, but still an issue with 404 on overview.view on the server

File: e2e/run.sh [hunk 1]
```diff
@@ -12,23 +12,31 @@
...
-exit $?
+  npx playwright install --with-deps # install browsers
+  npx playwright test
+
+  cd ..
+  cd docker
+  ./shutdown.sh 
+done
 
+exit $?
```
Comment (hmottestad) [inline e2e/run.sh:42]: What happens if the test fail for tomcat but not jetty? Does it still error out?

Comment (barthanssens) [reply e2e/run.sh:42]: hmz yes, that's on the TODO list  (and it probably doesn't hurt to update the docker documentation as well...)

Comment (barthanssens) [reply e2e/run.sh]: fixed

**Pull request merged**